tests/common/plugins/tacacs.py

```diff
@@ -17,6 +17,12 @@ def configure_tacacs(ptfhost, duthost, creds, tacacs_server_ip):
     duthost.shell("sudo config tacacs add %s" % tacacs_server_ip)
     duthost.shell("sudo config tacacs authtype login")
 
+    # option to connect with an external tacacs server
+    if external_tacacs_info['local_username']:
+        duthost.shell("sudo config tacacs add %s" % creds['tacacs_servers'][0])
+        duthost.shell("sudo config tacacs authtype pap")
+        duthost.shell("sudo useradd {} --password {}".format(external_tacacs_info['local_username'], external_tacacs_info['local_password']), module_ignore_errors=True)
```

Comment:
If we are testing with remote tacacs server, then "useradd" should not be done right? the username/password should verified at the server side.